app/src/utils/data/contributions.ts

```diff
@@ -361,7 +362,7 @@ export function grantDonationListener(
   // filter for every event by topic so that we can get to the event args too
   const filter = {
     address: GRANT_ROUND_MANAGER_ADDRESS,
-    topics: [id('GrantDonation(uint96,address,uint256,address[])')],
+    topics: [id('GrantDonation(uint96,address,uint256,address[],uint256)')],
```

Comment:
This adds the `time` arg to the topic filter 👍 - This is the main thing which was preventing the `GrantDonations` from updating.